web/components/feed/feed-comments.tsx

```diff
@@ -77,38 +94,105 @@ export function FeedCommentThread(props: {
   )
 }
 
-export const FeedComment = memo(function FeedComment(props: {
+export function ParentFeedComment(props: {
   contract: Contract
   comment: ContractComment
   showTip?: boolean
   myTip?: number
   totalTip?: number
-  indent?: boolean
+  seeReplies: boolean
+  numComments: number
   onReplyClick?: (comment: ContractComment) => void
+  onSeeReplyClick: () => void
 }) {
-  const { contract, comment, myTip, totalTip, showTip, indent, onReplyClick } =
-    props
   const {
-    text,
-    content,
-    userUsername,
-    userName,
-    userAvatarUrl,
-    commenterPositionProb,
-    commenterPositionShares,
-    commenterPositionOutcome,
-    createdTime,
-    bountiesAwarded,
-  } = comment
-  const betOutcome = comment.betOutcome
-  let bought: string | undefined
-  let money: string | undefined
-  if (comment.betAmount != null) {
-    bought = comment.betAmount >= 0 ? 'bought' : 'sold'
-    money = formatMoney(Math.abs(comment.betAmount))
-  }
-  const totalAwarded = bountiesAwarded ?? 0
+    contract,
+    comment,
+    myTip,
+    totalTip,
+    showTip,
+    onReplyClick,
+    onSeeReplyClick,
+    seeReplies,
+    numComments,
+  } = props
+  const { text, content, userUsername, userAvatarUrl } = comment
+
+  const router = useRouter()
+  const highlighted = router.asPath.endsWith(`#${comment.id}`)
+  const commentRef = useRef<HTMLDivElement>(null)
+
+  useEffect(() => {
+    if (highlighted && commentRef.current != null) {
+      commentRef.current.scrollIntoView(true)
+    }
+  }, [highlighted])
```

Comment:
(If you're curious, the bug is that on Nextjs pages rendered with `getStaticProps`, the router is not ready when it's first rendered.

We need to run an effect to check the path *after* the router is ready.

The reason for this is `getStaticProps` caches one rendered html response for the page. It serves the cached response to everyone so the page loads really fast. But, since there's only one cached version, it can't respond to different url params until after it has loaded.)

Reply:
Ohh interesting! Thanks James, I really appreciate you explaining these things, it helps me learn faster :)